### PR TITLE
Update betterlockscreen.sh

### PR DESCRIPTION
**Pull request by skandix**
added missing packages in the install script... installed on latest stable debian stretch

### CODE REVIEW

**Comment by skandix** [comment]
also this 
what_the_heck

**Comment by skandix** [comment]
didn't edit the packages localy...#lazy

**Comment by UtkarshVerma** [comment]
@skandix Thanks for the contribution, however this commit breaks support for my Lubuntu(bionic). I think conditionals for Debian Stretch would be better.

File: betterlockscreen.sh [hunk 1]
```diff
@@ -49,9 +49,9 @@ printf "\n";
 version=$(git ls-remote --tags https://github.com/pavanjadhaw/betterlockscreen | tail -1 | grep -o "v.*$");
 
 ## Dependencies
-sudo apt install bc imagemagick libjpeg-turbo8-dev libpam0g-dev libxcb-composite0 libxcb-composite0-dev \
-    libxcb-image0-dev libxcb-randr0 libxcb-util-dev libxcb-xinerama0 libxcb-xinerama0-dev libxcb-xkb-dev \
-    libxkbcommon-x11-dev feh libev-dev;
+sudo apt install bc imagemagick libjpeg-turbo62-dev libpam0g-dev libxcb-composite0 libxcb-composite0-dev \
+    libxcb-image0-dev libxcb-randr0 libxcb-xinerama0 libxcb-xinerama0-dev libxcb-xkb-dev \
+    libxkbcommon-x11-dev feh libev-dev dh-autoreconf libxcb-util0-dev;
```
Comment (UtkarshVerma) [inline betterlockscreen.sh:54]: Don't rename the packages, please. Implementing conditionals for Debian would be a good start.

**Review by UtkarshVerma — COMMENTED** [review]
Breaks Ubuntu support.

**Comment by UtkarshVerma** [comment]
Closing this due to inactivity.

**Pull request closed without merging**